src/OpenTelemetry/CHANGELOG.md

```diff
@@ -5,6 +5,9 @@
 * Increased the character limit of the Meter instrument name from 63 to 255.
   ([#4774](https://github.com/open-telemetry/opentelemetry-dotnet/issues/4774))
 
+* Update default size for `SimpleExemplarReservoir` to `1`.
```

Comment:
Add https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/CHANGELOG.md#:~:text=Experimental%20(pre%2Drelease%20builds%20only)%3A

Reply:
I think it's fine to not add this. We only want to add this when introducing a public API change.